Assets/Scripts/SpawnManager.cs

```diff
@@ -19,6 +19,8 @@ void Start()
 				SpawnPoints = GameObject.FindGameObjectsWithTag("Respawn");
 	            Vector3 spawnPositon = GetRandomSpawnPoint();
 	            LocalPlayer = PhotonNetwork.Instantiate("NetPlayer", spawnPositon, Quaternion.identity, 0);
+                // todo: Delete once event system is in place
+                LocalPlayer.GetComponent<LifeManager>().mDepositManager = GameObject.Find("DepositBox").GetComponent<DepositManager>();

```

Comment:
Why do you grab the DepositManager from the life manager versus just getting the deposit manager?